src/Compilers/CSharp/Test/Symbol/Symbols/DefaultInterfaceImplementationTests.cs

```diff
@@ -50163,9 +50168,6 @@ public sealed class TypeIdentifierAttribute : Attribute
         [Fact]
         public void NoPia_01()
         {
-            var attributes = CreateCompilation(NoPiaAttributes, options: TestOptions.ReleaseDll, targetFramework: TargetFramework.Net50);
-            var attributesRef = attributes.EmitToImageReference();
```

Comment:
This pattern was removed in the file because `TargetFramework.Net50` now includes all the standard references which brings in `System.Runtime.InteropServices.dll` which has this definition.